gm_dotnet_managed/Tests/PushCFuncPtr.cs

```diff
@@ -43,7 +43,7 @@ public Task<bool> Start(ILua lua, GetILuaFromLuaStatePointer lua_extructor, Modu
             {
                 unsafe
                 {
-                    lua.PushCFunction((delegate* unmanaged[Cdecl]<IntPtr, int>)(delegate* managed<IntPtr, int>)&TestFunc);
+                    lua.PushCFunction(&TestFunc);
```

Comment:
@AaronJThompson As I mentioned in previous PR, address-of operator now works properly with functions with unmanaged calling conventions.

Reply:
Nice! Looks much better